elements/pfe-styles/src/pfe-typography-deprecated.scss

```diff
@@ -0,0 +1,42 @@
+@import "../../pfe-sass/pfe-sass";
+
+// Deprecated header classes
+.pfe-headline-xxl  {
```

Comment:
If it helps, this might be a little more straight-forward:

```
@each $size in (xxl, xl, alpha, beta, gamma, delta, epsilon, zeta) {
  .pfe-headline-#{$size}  {
      @extend %pfe-headline-#{$size}; 
  }
}
```